### ISSUE TITLE
`flatten` loses `description` field

### ISSUE THREAD

**Issue by OliverJAsh** [unsplash/intlc#84]
```sh-session
$ cat app/components/AddToCollectionButton/lang/en-US.translations.json
{
  "button_title": {
    "message": "Add to collection",
    "description": "For accessibility, we set a title attribute on the button element that is shown when hovering over it"
  }
}

$ intlc flatten app/components/AddToCollectionButton/lang/en-US.translations.json
{"button_title":{"message":"Add to collection","backend":"ts"}}
```

It also adds `backend` but I don't think that does any harm.

**Comment by samhh** [comment]
The codebase currently ignores/doesn't know about `description`, or any other fields that might be present. The simplest solution is to make it aware and, if present, pass the field along, making this part of the "schema".

**Comment by samhh** [comment]
Fixed by: #85
